cluster/gce/gci/mounter/Dockerfile

```diff
@@ -14,6 +14,10 @@
 
 FROM ubuntu:xenial
 
-RUN apt-get update && apt-get install -y netbase nfs-common=1:1.2.8-9ubuntu12 glusterfs-client=3.7.6-1ubuntu1
+RUN apt-get update
+RUN apt-get install -y software-properties-common
+RUN add-apt-repository -y ppa:gluster/glusterfs-4.0
```

Comment:
According the "https://www.gluster.org/release-schedule/" said, the 4.0 version is EOL.
So the "ppa:gluster/glusterfs-4.1" repo is better.
https://launchpad.net/~gluster/+archive/ubuntu/glusterfs-4.1

Reply:
Sorry, I had thought I already did that. I've made that change, did a quick test, and pushed a commit with that adjustment. Thanks!